src/components/Login/Login.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Input } from "@material-ui/core";
+import { Button } from "@material-ui/core";
+import { Link, useNavigate } from "react-router-dom";
+import M from "materialize-css";
+import { useMedia } from "../../MediaContext";
+import { UserAction } from "../../action/userAction";
+import { useDispatch, useSelector } from "react-redux";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
```

Comment:
Even after logging in, the user can still access the login route which makes no sense, better to navigate it to some other route.